src/Microsoft.AspNet.Mvc.Localization/IHtmlLocalizer.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Localization;
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// Represents a service that provides localized HTML content. This service just encodes the arguments but not the 
```

Comment:
`This service does not HTML encode the resource string. It HTML encodes all arguments that are formatted in the resource string.`